<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Marketplace</class>
 <widget class="QWidget" name="Marketplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Marketplace</string>
  </property>
  <widget class="QLabel" name="walletLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>wallet</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="productTable">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="inventoryTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>231</width>
     <height>331</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>48</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Inventory</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatDisplay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>691</width>
     <height>75</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="chatInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>601</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>480</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
